incubator/hnc/go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/kubernetes-sigs/multi-tenancy/incubator/hnc
 go 1.12
 
 require (
+	contrib.go.opencensus.io/exporter/stackdriver v0.13.0
```

Comment:
Why wasn't this part of the last PR?

Reply:
It should be included in the last PR, but I excluded them.

Reply:
Thanks